tests/test_basic.py

```diff
@@ -74,9 +58,19 @@ def setup(self, request):
         else:  # weekend
             self.packets = 100000
 
-    def run_test(self, delay, device, ip, interface, library):
-        setup = f'{self.processes}_{self.ports}_{self.protocol}_{device}_{interface}'
-        binary = pathlib.Path(f'xtcp_bombard_{library.lower()}/bin/{setup}/xtcp_bombard_{library.lower()}_{setup}.xe')
+    def run_test(self, delay, target):
+        
+        if (target == 'xcore.ai'):
+            device = 'EXPLORER'
```

Comment:
seems odd that device is an (old) board name or an architecture name. I guess it should be "board" and its usage uniform?